ipmiMgrApp/Db/fru_cu_alias.template

```diff
@@ -0,0 +1,7 @@
+include "fru_cu.db"
```

Comment:
Isn't it [`fru_cu_common.db`][1] in the upstream?

I think, in the end, this is basically the same question I've made regarding `fru_basic_alias`, since `fru_cu.db` includes the substituted records from `sensor_ai.db` while `fru_cu_common.db` does not.

[1]: https://github.com/slac-epics-modules/ipmiComm/blob/R4.6.0/Db/fru_cu_common.db